Configuration/TypoScript/constants.txt

```diff
@@ -0,0 +1,15 @@
+plugin.tx_servicebw2_servicebw {
+  view {
+    # cat=plugin.tx_servicebw2_servicebw/file; type=string; label=Path to template root (FE)
```

Comment:
Please use `:` in Label to differ between Header and Description

Reply:
There is only a header for those items. Maybe looks wrong with the GitHub Editor